src/coreclr/md/compiler/regmeta.h

```diff
@@ -939,7 +939,7 @@ class RegMeta :
         mdTypeDef   td,                     // Parent TypeDef
         LPCWSTR     szName,                 // Name of member
         DWORD       dwFieldFlags,           // Member attributes
-        PCCOR_SIGNATURE pvSigBlob,          // [IN] point to a blob value of COM+ signature
+        PCCOR_SIGNATURE pvSigBlob,          // [IN] point to a blob value of CLR signature
```

Comment:
```suggestion
        PCCOR_SIGNATURE pvSigBlob,          // [IN] point to a blob value of signature
```
This would read better if it just said signature. "CLR signature" is not a thing.